BaseSpace.SDK/ServiceModels/Request/ListProjectsRequest.cs

```diff
@@ -7,6 +7,8 @@ public class ListProjectsRequest : AbstractResourceListRequest<ListProjectsRespo
     {
         public string Name { get; set; }
 
+        public string[] Include { get; set; }        
```

Comment:
Also the API supports the query param ?include= for GET: projects/{id}. Could you add the same field to GetProjectRequest?